WordPressKit/WordPressKit/Activity.swift

```diff
@@ -1,62 +1,66 @@
 import Foundation
 
-public struct RemoteActivity {
+public struct Activity {
     public let activityID: String
     public let summary: String
     public let name: String
     public let type: String
     public let gridicon: String
     public let status: String
     public let rewindable: Bool
-    public let published: Date?
-    public let actor: RemoteActivityActor?
-    public let object: RemoteActivityObject?
-    public let target: RemoteActivityObject?
-    public let items: [RemoteActivityObject]?
+    public let rewindID: String?
+    public let published: Date
+    public let actor: ActivityActor?
+    public let object: ActivityObject?
+    public let target: ActivityObject?
+    public let items: [ActivityObject]?
 
     init(dictionary: [String: AnyObject]) throws {
         guard let id = dictionary["activity_id"] as? String else {
             throw RemoteActivityError.missingActivityId
         }
+        guard let publishedString = dictionary["published"] as? String else {
+            throw RemoteActivityError.missingPublishedDate
```

Comment:
Since you removed the `Remote` prefix in the other types, maybe here as well? Or it could be nested inside `Activity` (as `Activity.Error`)